certbot/renewal.py

```diff
@@ -190,6 +191,15 @@ def _restore_required_config_elements(config, renewalparams):
                     raise errors.Error(
                         "Expected a numeric value for {0}".format(config_item))
             setattr(config.namespace, config_item, int_value)
+    # bool-valued items to add if they're present
```

Comment:
The test coverage of `renewal.py` is a mess for historical reasons (they're mostly in `certbot/tests/main_test.py`, and are more like functional tests than unit tests), but we should probably use each PR landing there as an opportunity to migrate towards consistent unit test coverage. So let's try to add a simple unit test for at least for the new `BOOL_CONFIG_ITEMS` logic, and maybe for all of `_restore_required_config_elements` to `certbot/tests/renewal_test.py` as part of this PR.

@thomaszbz, let us know whether you'd be comfortable tackling that, or might need guidance or help in writing unit tests.